README.md

```diff
@@ -28,6 +28,10 @@ It's helpful to build out an application step-by-step, optimizing each one. This
 
 ![Bitcoin Core App major versions](/images/bitcoin-core-app-versions.png)
 
+### Primary long-term goal
+
+To create a user friendly Bitcoin Core GUI targeted at storing and securing a users Bitcoin savings that is backed by their own node. This would also include an optional companion mobile signing app that can be part of a multi-sig setup. 
```

Comment:
The part about the signing app is a bit vague with the `would` and `can`. Feels like this should almost be a whole separate effort, since we're not even actively working on this. What do you think?

Reply:
I think that should be the main focus of the mobile aspect of the Bitcoin Core app. Running a node and not using it isn't very useful.

Reply:
Initially, we had 3 phases to this application with the middle one being about key-signing functionality. IIRC, we removed it because the group felt that it should be a separate app and actually slow down development. Was that your understanding also? Lets ensure we're all on the same page about this. If it is a separate app, then it could be an altogether different effort, using a different tech stack, built by different people, etc.

Reply:
Maybe it isn't so clear, when we saying key signing functionality do we mean more complex key-signing operations like multisig? Not just regular local signing which all wallets do? 

To me, the main value proposition of the GUI should be being a secure cold-storage wallet (vault?) for your bitcoin savings. Part of that security being that it is backed by and uses your own local node - never a third party. Running a node is more suited for desktop from previous discussions. The only role a mobile device should play in a secure cold-storage setup is to be a watch-only wallet / external signer and maybe a node remote. Having a mobile run node could  be an option but should be more of a _'nice to have'_ for more marginal use cases and not a priority like it currently is.

I'm leaning more towards Steve's concerns that running a node on mobile may not be the best application of resources. I think there is whole lot more value in building what I mentioned above and then _maybe_ building a companion mobile app that can be used as an external signer and/or node remote. Ideally any external Bitcoin wallet should be able to be an external signer for the GUI using PSBTs. Interoperability should be a bigger focus than mobile.

Below is some stages:

1. Build a QML desktop node runner.
    - Our current focus but prioritize desktop. Would be great to have some nice visuals / block analysis as part of this. mempool.space being an example. 
    - I resonate with Steve's post on Slack that watching and knowing you are verifying blocks yourself is very satisfying / liberating - could get creative here ;)  
2. Add basic wallet functionality (Local wallets / PSBT - same as what's in the widgets GUI currently).
3. Add local only multisig (think Nunchuk / Specter at this stage). 
4. Focus on making this desktop GUI interoperable with other wallets in the ecosystem. 
    - Say if you wanted to have 1 signer in BlueWallet and 1 in the Bitcoin Core desktop app. 
5. Stretch goal: Build a Bitcoin Core mobile companion app that acts as an external signer / node remote / possibly a node runner. 

Another nice to have could be if you could use applications with your local node. Say launch your own localhost mempool.space instance using your node - this would need some kind of 'app' store.

Reply:
By key-signing, I mean a UI purely focused on managing keys and signing transactions, like [this old concept](https://github.com/GBKS/bitcoin-design/blob/main/shared-images/Bitcoin%20key%20locker%20200914.png). Use case is for external signing and multi-key setups. Goal is to make that use case super simple, and also more accessible by turning any phone into a signer. My understanding is that this will not be part of this project, so I see it as a completely separate effort (one that no-one is working on at the moment).

Overall, I'd prefer to focus on the minimal experience and super-tight execution before moving to anything complex or addressing nice-to-haves. Might be a little slower at the start, but should be beneficial long-term. So I'd start with ensuring this application works smoothly across operating systems, we have smooth onboarding, and there's a fallback for phones that can't run the node, like connecting to a remote node. If possible, I'd also like to see dock integration on desktops, like [this concept](https://github.com/GBKS/bitcoin-design/blob/main/shared-images/Bitcoin%20Node%20Control%20Center%20200721.png). I expect that once we actually start using this, we'll have a better idea of what to build next.

Reply:
Can we resolve that one? I'd not hint at a second project (key signer) as nobody is seriously thinking about it or working on it. Keeping it simple for now.

Reply:
Made several updates:

- Implemented suggestions above.
- Updated Figma source files.
- Clarified the copy.